scene/gui/item_list.cpp

```diff
@@ -369,6 +369,7 @@ void ItemList::clear() {
 	update();
 	shape_changed = true;
 	defer_select_single = -1;
+	scroll_bar->set_value(0);
```

Comment:
`scroll_bar`'s `value_changed` signal is connected to a method that calls `update()`, so we're calling it twice here now. Not sure how to prevent it, or whether it's worth the trouble - just mentioning it for the review.